docker/Dockerfile.ci_cortexm

```diff
@@ -45,8 +45,8 @@ ENV PYTHONNOUSERSITE 1  # Disable .local directory from affecting CI.
 COPY install/ubuntu_install_python_package.sh /install/ubuntu_install_python_package.sh
 RUN bash /install/ubuntu_install_python_package.sh
 
-COPY install/ubuntu1804_install_llvm.sh /install/ubuntu1804_install_llvm.sh
-RUN bash /install/ubuntu1804_install_llvm.sh
+COPY install/ubuntu2004_install_llvm.sh /install/ubuntu2004_install_llvm.sh
+RUN bash /install/ubuntu2004_install_llvm.sh
```

Comment:
@mehrdadh, thanks for this upgrade. Since Ubuntu20.04 has default installation of `python3.8` and `python3.7` EOL is June 2023 as per https://peps.python.org/pep-0537/#lifespan. Since I have not seen any other discussions, my assumption is that we still do wish to support older version until its EOL. We most likely would require old installation from https://launchpad.net/~deadsnakes/+archive/ubuntu/ppa.

Reply:
@ashutosh-arm do we need python3.7 for cortexm CI image? If so, we should add it in a separate PR.
This PR doesn't update the image yet.

Reply:
Yes, we will need to support `python3.7` until its EOL. Considering this, the next thing that pops up in my brain is when the `ci_cortexm` gets updated eventually corresponding to this change, will the default venv still remain at python3.7 or we would move cortex-m docker image to use `install/ubuntu2004_install_python.sh`? https://github.com/apache/tvm/blob/92da138bcb18c76604763785d3177de3f9f9b296/docker/Dockerfile.ci_cortexm#L38

Reply:
`install/ubuntu2004_install_python.sh` installs python3.8 as well. We have to add another script to install 3.7 in ubuntu20.04 if we need it.

Switching the ci image to 3.8 doesn't mean we're not supporting 3.7 anymore. Could you elaborate if there is a specific reason that this image has to run 3.7? We have switched other images like hexagon to 3.8 already.

Reply:
I see your point, I will change the scripts to install python3.7 in ci_cortexm.

Reply:
https://github.com/apache/tvm/pull/13736/commits/08ce7f8086d6fa6ec3281eab9b28663d269f0cbb addresses your concern.

Reply:
Thanks @mehrdadh